packages/manager/modules/pci/src/projects/project/data-processing/job-details/job-dashboard/job-dashboard.module.js

```diff
@@ -4,7 +4,7 @@ import '@uirouter/angularjs';
 
 import routing from './job-dashboard.routing';
 
-import PciStoragesContainersService from '../../../storages/containers/containers.service';
+import PciStoragesContainersService from '../../../storages/object-storage/containers/containers.service';
```

Comment:
I'm having trouble understanding why `containers` which is generic falls under specific product `object-storage` as it is also used for `cloud-archive` for instance

Reply:
oh, i thought only the component is being used in cloud-archive. So container becomes a generic module between object storage and cloud archive ?

I have to refactor this again

Reply:
Hi @marie-j, Please have a look now. i was a bit confused about refactoring this. This should be okay

Reply:
Maybe we can sync so we're on the same page ? :) It will avoid to refactor too much